### PR TITLE
[script.embuary.helper] 2.0.0

### PR DESCRIPTION
**Pull request by sualfred**
### Description

Dropped Py2 support

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.embuary.helper/resources/lib/player_monitor.py [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+        if method == 'Player.OnAVChange':
+            self.get_audiotracks()
+
+            if self.pvr_playback:
+                self.get_art_info()
+
+        ''' Playback stopped. Clean up.
+        '''
+        if method == 'Player.OnStop':
+            xbmc.sleep(2500)
```
Comment (enen92) [inline script.embuary.helper/resources/lib/player_monitor.py]: Still fine as 2500 is quite small but would be better to use the monitor

File: script.embuary.helper/resources/lib/utils.py [hunk 2]
```diff
@@ -0,0 +1,736 @@
...
+            value = True
+        elif value.lower() == 'false':
+            value = False
+
+    json_call('Settings.SetSettingValue',
+              params={'setting': '%s' % settingname, 'value': value}
+              )
+
+
+def toggleaddons(params):
```
Comment (enen92) [inline script.embuary.helper/resources/lib/utils.py]: I probably asked before but what is this for? 
Not sure if isn't better to create some sort of whitelist to only allow the execution of this method provided a list of allowed addons. Still, we have this exposed in json anyway...

File: script.embuary.helper/resources/lib/utils.py [hunk 3]
```diff
@@ -0,0 +1,736 @@
...
+
+
+def txtfile(params):
+    prop = params.get('prop')
+    path = xbmc.translatePath(remove_quotes(params.get('path')))
+
+    if os.path.isfile(path):
+        log('Reading file %s' % path)
+
+        file = open(path, 'r')
```
Comment (enen92) [inline script.embuary.helper/resources/lib/utils.py]: use with

File: script.embuary.helper/addon.xml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.embuary.helper" name="Embuary Helper" version="2.0.0" provider-name="sualfred">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+        <import addon="script.module.pil" version="1.1.7"/>
+    </requires>
+    <extension point="xbmc.python.script" library="default.py">
+        <provides>executable</provides>
+    </extension>
+    <extension point="xbmc.python.pluginsource" library="plugin.py">
```
Comment (enen92) [inline script.embuary.helper/addon.xml]: I'm not really a fan of having a plugin for a script named an helper. The url `plugin://script.embruary.helper` sounds like an hack. Can't you refactor the plugin part somewhere else?

File: script.embuary.helper/addon.xml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.embuary.helper" name="Embuary Helper" version="2.0.0" provider-name="sualfred">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+        <import addon="script.module.pil" version="1.1.7"/>
```
[the remark this answers is not in the document]
Comment (sualfred) [reply script.embuary.helper/addon.xml]: Already in Matrix, yes.

File: script.embuary.helper/resources/lib/utils.py (hunk 2 above)
Comment (sualfred) [reply script.embuary.helper/resources/lib/utils.py]: Is allowed. Asked in the past.

File: script.embuary.helper/addon.xml (hunk 4 above)
Comment (sualfred) [reply script.embuary.helper/addon.xml]: Very common and used by a lot of addons. Maybe I'll split it up into two different addons in the future. But no time for it atm.

File: script.embuary.helper/resources/lib/utils.py (hunk 3 above)
Comment (sualfred) [reply script.embuary.helper/resources/lib/utils.py]: WIll change it

File: script.embuary.helper/resources/lib/player_monitor.py (hunk 1 above)
Comment (sualfred) [reply script.embuary.helper/resources/lib/player_monitor.py]: switched to monitor

File: script.embuary.helper/resources/lib/utils.py (hunk 3 above)
Comment (sualfred) [reply script.embuary.helper/resources/lib/utils.py]: changed

**Pull request merged**